app/uk/gov/hmrc/eusubsidycompliancefrontend/connectors/Connector.scala

```diff
@@ -63,14 +63,14 @@ trait Connector extends TracedLogging {
     url: String,
     payload: A
   )(implicit hc: HeaderCarrier, ec: ExecutionContext, wts: Writes[A]): ConnectorResult = {
-    logger.info(s"$className.$methodName - posting to $url with $payload")
+    logger.info(s"$className.$methodName - posting to $url")
     makeRequest(_.POST[A, HttpResponse](url, payload)).map {
       case Left(error) =>
-        logger.error(s"$className.$methodName - failed POST to $url with $payload", error)
+        logger.error(s"$className.$methodName - failed POST to $url", error)
         Left(error)
       case Right(successResponse: HttpResponse) =>
         logger.info(
-          s"$className.$methodName - successful POST to $url with $payload returned ${successResponse.body}"
+          s"$className.$methodName - successful POST to $url"
```

Comment:
This stuff was just used for the EIS stuff. No emails etc. Still risky. Just means if we have problems with ETMP etc we may get dragged into drawn out meetings trying to work out what the problems were :(